generate/generate_nircam.py

```diff
@@ -558,43 +558,30 @@
                     for sca in ["1", "2", "3", "4", "5"]
                 ],
                 [
-                    "NRCA2_MASK210R",
-                    "NRCA5_MASK210R",
-                    "NRCA2_TAMASK210R",
-                    "NRCA2_FSTAMASK210R",
+                    "NRCA5_{}STRIPE{}_DHS_F322W2".format(pix,stripe)
```

Comment:
Fancy -- A little less readable in my opinion but if this is a request added by a team I won't say anymore 👍